### DISCUSSION TITLE
BLD Boost for Vesting BLD?

### DISCUSSION THREAD

**Discussion by erichitchwood** [Agoric/agoric-sdk#5723] [Q&A] [answered]
Can those with BLD locked in vesting contracts borrow IST against their future staking rewards with BLD Boost? Can they freely swap borrowed IST into USDC via the PSM?

**Comment by dckc** [comment, marked as answer]
**Yes**, once BLD boost is launched as part of Inter Protocol, if you can stake your BLD, you can borrow IST against it. (And tokens locked in a periodic vesting contract can be staked.) You can then swap to high-quality stable tokens such as USDC via the PSM.

> 2.2 BLD Boost
> ... Holders of the BLD token that stake their BLD with an active validator can lock up a
> small portion of their staked BLD and mint IST against it.
-- [The Inter Protocol Whitepaper](https://inter.trade/static/whitepaper-55bdca7d060f4e2618eae4db34765f9d.pdf) Version 0.93 June 20, 2022
